css/style.css

```diff
@@ -16,4 +16,278 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+/* Header - logo and nav */
+
+.header {
+    display: flex;
+    justify-content: space-around;
+    padding-top: 30px;
+}
+
+.header-logo {
+    margin-right: 400px;
+}
+
+.logo {
+    height: 75px;
+}
+
+.header-nav {
+    display: flex;
+    justify-content: space-between;
+    width: 40%;
+    -ms-transform: translateY(50%);
+    transform: translateY(50%);
+}
+
+.header-nav-link {
+    text-decoration: none;
+    font-size: 120%;
+    color: darkslategray;
+    opacity: 70%;
+}
+
+.header-nav-link:hover {
+    color: coral;
+    opacity: 1;
+}
+
+.header-nav-link:first-child {
+    opacity: 1;
+}
+
+/* Section 1 */
+
+.section1 {
+    background-image: url("/img/first-background.jpg");
+    background-color: white;
+    height: 1000px;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    position: relative;
+    text-align: center;
+    color: white;
+    font-size: 200%;
+    font-weight: lighter;
+}
+
+.section1-article {
+    -ms-transform: translateY(115%);
+    transform: translateY(115%);
+}
+
+.section1-title {
+    font-weight: lighter;
+}
+
+.section1-text {
+    font-weight: lighter;
+    margin-bottom: 75px;
+}
+
+.section1-button {
```

Comment:
I think here as well you can use the same class for all buttons and add an id or class for a different style.
exp : <button class="orange--button" id="section3-button" >
`.orange---button :{
  background-color: coral;
    border: none;
    border-radius: 5px;
    font-size: 80%;
    padding: 20px 40px;
}
#section-button:{ 
color: white;
    text-decoration: none;}`

Reply:
Another bad habit, I rarely complete all the html before I start styling. I think this is what leads to what you've pointed out. Thanks again for pointing it out, it's great to get feedback.